index.bs

```diff
@@ -112,6 +112,7 @@ interface Socket {
   Promise&lt;undefined> close(optional any reason);
 
   readonly attribute boolean upgraded;
+  readonly attribute String secureTransport;
```

Comment:
There's an existing enum defined in the spec for this. https://sockets-api.proposal.wintertc.org/#enumdef-securetransportkind